pyproject.toml

```
@@ -161,4 +161,10 @@ inline-quotes = "single"
 "docs/sphinxext/github_link.py" = ["BLE001"]
 
 [tool.ruff.format]
-quote-style = "single"
\ No newline at end of file
+quote-style = "single"
+[tool.codespell]
```

Just to keep it consistent with the above sections:

```suggestion

[tool.codespell]
```

---

sure, thanks!